.gitattributes

```diff
@@ -0,0 +1 @@
+*    text eol=lf
```

Comment:
Is this still required? Looks like it was one of the iterative changes but the actual fix is rather in the additional upgrade step in the Dockerfile.

Reply:
It's not required for this fix although nice to have for windows users. I've reverted the commit so only the dockerfile changes are present in the PR now 👍